apps/strapi-cms/.env.default

```diff
@@ -15,3 +15,10 @@ AWS_BUCKET_ENDPOINT=
 AWS_BUCKET_NAME=
 AWS_BUCKET_FULL_URL=
 CDN_URL=
+
+# UPDATE STATIC CONTENT PLUGIN
+USC_GITHUB_PAT=
+USC_REPO_OWNER=
+USC_REPO_NAME=
+USC_WORKFLOW_ID=
+USC_TARGET_BRANCH=
```

Comment:
Why do you think the `USC` prefix is valuable?

Reply:
For two reasons: first of all the prefix lets the user identify the constants related to the USC plugin, secondary for consistency with the constants of AWS plugin

Reply:
You are not convincing me 😄 I mean, there are variable that, IMHO, should be general, without the need to specify the plugin or whatever uses it. It's an environment variable, not a plugin variable; that's why I'd call them differently.
E.g. `REPO_OWNER` instead of `USC_REPO_OWNER`.

The name should be as clear as possible and describe what piece of information that variable bring to the application: e.g. `USC_WORKFLOW_ID` could be something like `DEPLOY_WEBSITE_WORKFLOW_ID`, which tells me that variable contains the id of the `Deploy Website` workflow.